src/style.css

```
@@ -1 +1,59 @@
 /* add styles here */
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+/* First stars block */
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+}
```

```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background: url(images/star.svg) no-repeat center;
  margin-right: 4px;
}

```

Let's write like this and combine all styles